### PR TITLE
Add /manage

### PR DESCRIPTION
**Pull request by MahdiMds-Sholex**

### CODE REVIEW

File: LilSholex/groupguard/keyboards.py [hunk 1]
```diff
@@ -174,3 +174,41 @@ def inlinePanel(chat_id, from_id):
              {'text': group.database.inline_keyboard_lock, 'callback_data': f'change_inline_keyboard_lock|{chat_id}'}]
         ]}
     }
+
+def managePanel(chat_id, reply_id):
```
Comment (AntiWanted) [inline LilSholex/groupguard/keyboards.py:178]: Use snake_case here

File: LilSholex/groupguard/keyboards.py [hunk 2]
```diff
@@ -174,3 +174,41 @@ def inlinePanel(chat_id, from_id):
...
+    user = classes.User(reply_id)
+    group = classes.Group(user.database, chat_id)
+    muted_text = {'en': 'Mute',
+                  'fa': 'بی صدا'}
+    if not group.get_chat_member(
+        reply_id
+    ).get('can_send_messages', None):
+        muted_text = {'en': 'Unmute',
```
Comment (AntiWanted) [inline LilSholex/groupguard/keyboards.py:186]: Move these texts to translators.py

File: LilSholex/groupguard/keyboards.py [hunk 3]
```diff
@@ -174,3 +174,41 @@ def inlinePanel(chat_id, from_id):
              {'text': group.database.inline_keyboard_lock, 'callback_data': f'change_inline_keyboard_lock|{chat_id}'}]
         ]}
     }
+
+def managePanel(chat_id, reply_id):
+    user = classes.User(reply_id)
+    group = classes.Group(user.database, chat_id)
```
Comment (AntiWanted) [inline LilSholex/groupguard/keyboards.py:180]: Pass the group from parent function 
Do not create a new group object here !

File: LilSholex/groupguard/views.py [hunk 4]
```diff
@@ -949,6 +1019,26 @@ def webhook(request):
                     keyboard
                 )
                 group.send_message(group.translate('panel_sent'), reply_to_message_id=message_id)
+            elif text == 'manage':
+                if group.get_chat_member(
+                        message['reply_to_message']['from']['id']
+                )['status'] not in ('creator', 'administrator'):
+
+                    keyboard = keyboards.managePanel(message['chat']['id'], message['reply_to_message']['from']['id'])
```
Comment (AntiWanted) [inline LilSholex/groupguard/views.py:1027]: Use snake_case for function names

**Comment by MahdiMds-Sholex** [comment]
Ok !

File: LilSholex/groupguard/keyboards.py (hunk 1 above)
Comment (MahdiMds-Sholex) [reply LilSholex/groupguard/keyboards.py]: Changed

File: LilSholex/groupguard/keyboards.py (hunk 2 above)
Comment (MahdiMds-Sholex) [reply LilSholex/groupguard/keyboards.py]: Moved

**Pull request merged**